lib/backend/k8s/conversion.go

```diff
@@ -63,13 +63,32 @@ func (c converter) parseWorkloadID(workloadID string) (string, string) {
 	return splits[0], splits[1]
 }
 
-// parsePolicyName extracts the Namespace and NetworkPolicy name from the given Policy name.
-func (c converter) parsePolicyName(name string) (string, string) {
-	splits := strings.SplitN(name, ".", 2)
+// parsePolicyNameNamespace extracts the Kubernetes Namespace that backs the given Policy.
+func (c converter) parsePolicyNameNamespace(name string) (string, error) {
+	// Policy objects backed by Namespaces have form "ns.projectcalico.org/<ns_name>"
+	if !strings.HasPrefix(name, "ns.projectcalico.org") {
```

Comment:
Include the / here please: avoids bad behavior if someone wants to name policies sillyly.